specification/mcpp/resource-manager/Microsoft.CloudPartnerProgramMembership/preview/2023-02-09-preview/mcpp.json

```diff
@@ -0,0 +1,447 @@
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "McppPurchaseDetailsResponse":{
+      "description": "MCPP Purchase details response",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "id for the purchase resource",
+          "readOnly": true
+        },
+        "name": {
+          "description": "purchase resource name.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "type": {
+          "description": "Resource type, CloudPartnerProgramMembership.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "properties": { 
+          "$ref": "#/definitions/McppPurchaseDetailsResponseProperties",
+          "description": "MCPP Purchase details response properties."
+        }
+      }
+    },
+    "McppPurchaseDetailsResponseProperties": {
+      "description": "MCPP Purchase details response properties",
+      "type": "object",
+      "properties": {
+        "orderNo": {
+          "type": "string",
+          "description": "order number for the purchase",
+          "readOnly": true
+        },
+        "orderDateTime": {
+          "type": "string",
+          "description": "date time at which the order was placed",
+          "readOnly": true
+        },
+        "status": {
+          "type": "string",
+          "description": "purchase status or the order",
+          "readOnly": true
+        },
+        "operationDetails":{
+          "type": "object",
+          "description": "operation details related to the purchase",
+          "readOnly": true
+        }
+      }
+    },
+    "CloudPartnerProgramMembershipPurchaseDetailsListResponse": {
+      "description": "Get purchase details of a resource",
+      "type":"object",
+      "properties": {
+        "value": {
+          "description": "The list of McppPurchaseDetailsResponse.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/McppPurchaseDetailsResponse"
+          }
+        }
+      }
+    },
+    "OperationListResult": {
+      "description": "Result of the request to list operations. It contains a list of operations and a URL link to get the next set of results.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          },
+          "x-ms-identifiers":[],
+          "description": "List of operations."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of operation list results if there are any."
+        }
+      }
+    },
+    "Operation": {
```

Comment:
>Operation [](http://example.com/codeflow?start=5&length=9)

leverage the common type defined here - [https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/types.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/types.json)

Reply:
ack, will make this change.